### PR TITLE
RHCLOUD-2050 - Copy 5xx reqest id to Sentry

### PR DESCRIPTION
**Pull request by tahmidefaz**

### CODE REVIEW

**Comment by tahmidefaz** [comment]
This could be very far away from what we want, I just want some feedback here

**Comment by tahmidefaz** [comment]
Also, I am not sure how we will be getting the `request_id` since different APIs seems to have different `request_id` headers

File: packages/utils/src/interceptors.js [hunk 1]
```diff
@@ -30,6 +30,16 @@ export function interceptor401(error) {
     throw error;
 }
 
+export function interceptor500(error) {
+    if (error.response && error.response.status === 503) {
```
Comment (karelhala) [inline packages/utils/src/interceptors.js]: The ACK might have been too fast, can we catch all 500 errors here? `if (error.response && error.response.status >= 503) {`

**Comment by ryelo** [comment]
I guess this is a broader question:

Is this assuming that apps are going to catch their own 500 errors and then use this interceptor to send the req id to sentry? I think the answer to that question is yes, but I want to make sure.

File: packages/utils/src/interceptors.js (hunk 1 above)
Comment (tahmidefaz) [reply packages/utils/src/interceptors.js]: @karelhala will saying `500` catch all `5xx` errors? or do I have to mention them individually?

**Comment by tahmidefaz** [comment]
@ryelo yeah, right now, that's correct! Do you think we will rather do something different here?

**Comment by ryelo** [comment]
@tahmidefaz This is good! Just do @karelhala's comment and it works for me!

**Comment by tahmidefaz** [comment]
@karelhala how about now?

File: packages/utils/src/interceptors.js [hunk 2]
```diff
@@ -30,6 +30,16 @@ export function interceptor401(error) {
     throw error;
 }
 
+export function interceptor500(error) {
+    if (error.response && error.response.status === 500) {
```
[the remark this answers is not in the document]
Comment (ryelo) [reply packages/utils/src/interceptors.js]: `     if (error.response && error.response.status >= 500 && error.response.status < 600) { `

[the remark this answers is not in the document]
Comment (ryelo) [reply packages/utils/src/interceptors.js]: If we want to just scope to 5xx errors I think something like the above should work

[the remark this answers is not in the document]
Comment (tahmidefaz) [reply packages/utils/src/interceptors.js]: I missed the `>=` :expressionless: 
also, what the.. didn't know you can do that with `.js`, crazy! 
I don't think there is any error greater than `5xx` :thinking:

[the remark this answers is not in the document]
Comment (tahmidefaz) [reply packages/utils/src/interceptors.js]: doing it anyway for the sake of custom Http status codes

**Pull request merged**